src/layout/DropZone.tsx

```diff
@@ -0,0 +1,130 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import React, { useCallback, useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { MdCloudUpload } from 'react-icons/md';
+
+export default function DropZone(props: {
+  color?: string;
+  children?: JSX.Element;
+}) {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [alert, setAlert] = useState('');
+  const [active, setActive] = useState(props.children ? true : false);
+  const onDrop = useCallback((acceptedFiles) => {
+    // Do something with the files
+    acceptedFiles.forEach((file: Blob) => {
+      const reader = new FileReader();
+
+      reader.onabort = () => setAlert('file reading was aborted');
+      reader.onerror = () => setAlert('file reading has failed');
+      reader.onload = () => {
+        setActive(true);
+        // Do whatever you want with the file contents
+        //const binaryStr = reader.result;
+        setAlert('file uploaded');
+        //console.log(binaryStr);
+      };
+      reader.readAsArrayBuffer(file);
+    });
```

Comment:
we should not read the files here. the component should accept a callback prop `onDrop: (files: TypeOfAcceptedFiles) => void`

Reply:
added